XamMacSwiftUITest.FSharp/AppDelegate.fs

```diff
@@ -10,11 +10,11 @@ open SwiftUI
 [<Register("AppDelegate")>]
 type AppDelegate() =
     inherit NSApplicationDelegate()
-    override self.ApplicationShouldTerminateAfterLastWindowClosed(_) = GC.Collect(); false
+    override self.ApplicationShouldTerminateAfterLastWindowClosed(_) = GC.Collect(); true
```

Comment:
This should ensure the app actually closes down, when we exit.

Reply:
Actually, I had it returning false so that the app didn't exit before the GC ran so that we can debug that.